static/js/lib/websocket.js

```diff
@@ -20,6 +20,14 @@ define([
      */
     var WebsocketManager = function (User, settings) {
             this.settings = $.extend(true, {}, this.settings, settings);
+            //Domain name to be changed
+            if(location.host.split(":").length > 1){
+                this.settings.serverUrl = "ws://"+location.hostname+":5000";                
```

Comment:
D'ailleurs les settings sont maintenant passés comme ça (voir chat.js)

js`this.settings  = $.extend(true, {}, this.settings, module.config(), settings);`

Donc plus besoin de définir "settings" dans le prototype